examples/ct/structures_test.go

```diff
@@ -106,21 +89,25 @@ func TestSerializeLogEntry(t *testing.T) {
 }
 
 // Creates a mock key manager for use in interaction tests
-func setupMockKeyManager(ctrl *gomock.Controller, toSign []byte) *crypto.MockKeyManager {
-	mockKeyManager := setupMockKeyManagerForSth(ctrl, toSign)
-	mockKeyManager.EXPECT().GetRawPublicKey().AnyTimes().Return([]byte("key"), nil)
+func setupMockPrivateKeyManager(ctrl *gomock.Controller, toSign []byte) *crypto.MockPrivateKeyManager {
+	pubkey, err := crypto.PublicKeyFromPEM(ctTesttubePublicKey)
+	if err != nil {
+		panic(err)
```

Comment:
Panicking, even in tests, is generally prohibited. Issues include that it causes tabular tests to abort early. Just return the error and fail the test.